src/content/apis/movies/IMDb.md

```diff
@@ -0,0 +1,10 @@
+---
+name: IMDb
+description: Access movie and TV information similar to the official one. Get Title, Year, Metascore Rating, IMDB rating, Release date, Runtime, Genre, Directors, Writers, Actors, Plot, Awards, Posters,etc…
+categories:
+  - movies
+features:
+  https: true
+url: https://imdb-api.com/api
+logo: https://seeklogo.com/images/I/imdb-logo-AF81176825-seeklogo.com.jpg

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.